pyomo/contrib/fbbt/tests/test_fbbt.py

```diff
@@ -727,33 +727,24 @@ def test_inf_bounds_on_expr(self):
         self.assertEqual(lb, None)
         self.assertEqual(ub, None)
 
-    @unittest.skip('This test passes locally, but not on travis or appveyor. I will add an issue.')
```

Comment:
I'm guessing this is a sneaky bug fix?

Reply:
It is actually a fix to the implementation of the test (leveraging the `LoggingIntercept` context manager resolves portability problems)